packages/xo-server/src/xo-mixins/metadata-backups.js

```diff
@@ -147,14 +147,15 @@ export default class metadataBackup {
     this._app = app
     this._logger = undefined
     this._runningMetadataRestores = new Set()
+    this._poolMetadataTimeout = parseDuration(backup.poolMetadataTimeout)
 
     const debounceDelay = parseDuration(backup.listingDebounce)
     this._listXoMetadataBackups = debounceWithKey(
       this._listXoMetadataBackups,
       debounceDelay,
       remoteId => remoteId
     )
-    this.__listPoolMetadataBackups = debounceWithKey(
+    this._listPoolMetadataBackups = debounceWithKey(
```

Comment:
Why this change?!

Reply:
To not create a PR for  deleting `_`.

Reply:
This has nothing to do with change, do another PR.